jaeger-core/src/main/java/com/uber/jaeger/SpanContext.java

```diff
@@ -35,6 +35,7 @@
 
   private final long traceId;
   private final long spanId;
+  // for zipkin compatibility
```

Comment:
this feels out of place. If we want to elaborate on the need for parentId, I'd move it to the class comment and make it more descriptive.